### ISSUE TITLE
Synonyms for entities are ignored.

### ISSUE THREAD

**Issue by Johnz86** [samtecspg/articulate#226]
Articulate v0.9.2

### Issues
> *User defined synonyms for entity value are ignored. Because of this user needs to create a larger data set to match any intent.*

 "entity_synonyms": [] are never filled for training_data.json rasa, even if user defines them in entity view.
![image](https://user-images.githubusercontent.com/2708520/40614548-81109860-6283-11e8-9bda-6fd1ebdf1c2a.png)

And training data will look like:
```{
  "rasa_nlu_data": {
    "common_examples": [
....
large list of intents
...
],
    "entity_synonyms": [],
    "regex_features": []
  }
```

**Comment by wrathagom** [comment]
Good catch, we didn't use these before as we were creating `common_examples` from a Cartesian product of the entities/synonyms and training examples. Now that we allow you to turn that functionality off We need to make sure and populate this.

**Comment by wrathagom** [comment]
@dcalvom I don't think there is any disadvantage to generating this section even when we are generating the Cartesian product.

**Comment by dcalvom** [comment]
Ok this is an easy one. I will tackle it tomorrow if you don't go for it first :)
